package.json

```diff
@@ -40,7 +42,8 @@
     "prisma": "^5.1.1",
     "supertest": "^6.3.3",
     "ts-jest": "^29.1.1",
-    "typeorm": "^0.3.17"
+    "typeorm": "^0.3.17",
+    "typescript": "^5.2.2"
```

Comment:
Do we want to fix the version of `typescript` we use and proceed to explicit upgrades?

Reply:
I feel like this is a separate question.